application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -146,6 +132,40 @@ private boolean isUserOnCooldown(User user) {
             .isPresent();
     }
 
+    private void sendCooldownResponse(SlashCommandInteractionEvent event) {
+        User user = event.getUser();
+
+        String message =
+                """
+                        Sorry, you can only create a single help thread every 5 minutes. Please use your existing thread %s instead.
+                        If you made a typo or similar, you can adjust the title using the command %s and the category with %s :ok_hand:""";
+
+        RestAction<String> changeTitle = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
+                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
+                HelpThreadCommand.CHANGE_TITLE_SUBCOMMAND);
+        RestAction<String> changeCategory = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
+                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
+                HelpThreadCommand.CHANGE_CATEGORY_OPTION);
+        long lastCreatedThreadId = database
+            .read(context -> context.selectFrom(HELP_THREADS)
+                .where(HELP_THREADS.AUTHOR_ID.eq(user.getIdLong()))
+                .orderBy(HELP_THREADS.CREATED_AT.desc())
+                .fetch())
+            .get(0)
+            .getChannelId();
+
+        if (lastCreatedThreadId == 0) {
+            logger.warn("Can't find the last help thread created by the user with id ({})",
```

Comment:
give some context, add what you attempted to do.

Reply:
it already does i said can't find the last help thread and gave the user id

Reply:
yeah but i meant the context of the command. when u read that message, u dont know that its:
* ask command
* on cooldown

Reply:
can u give a fully constructed sentence, im bad at these things 😅

Reply:
Sure

> Attempted to find the last help thread of user {} during ask-cooldown, but there is none.